wordpress-mysql-replication/website.json

```diff
@@ -138,7 +138,7 @@
                         "[concat('Microsoft.Web/Sites/', parameters('siteName'), '/config/web')]"
                     ],
                     "properties": {
-                        "packageUri": "https://auxmktplceprod.blob.core.windows.net/packages/wordpress-4.2.4-IIS.zip",
+                        "packageUri": "https://auxmktplceprod.blob.core.windows.net/packages/wordpress-4.4.2-IIS.zip",
```

Comment:
Is this a special package? Can this not be downloaded from an official public location rather than somebody's storage account for transparency?